config.js

```diff
@@ -140,7 +140,6 @@ const clockJobConfig = {
   intervals: {
     checkMissedCollectorHeartbeat:
       String(collectorConfig.heartbeatIntervalMillis),
-    deactivateRooms: '5m',
```

Comment:
Removing this from the list is enough to skip the JOB

Reply:
I think we should be leaving all the code here, since it is still a valid use case, even though we don't want to use it in our instance, so we should just set overrides in config which would effectively disable the job

Reply:
Keeping all the code, added env var ENABLE_DEACTIVATE_ROOMS - this will be false and inside setupIntervals this will get false under toggles for deactivateRooms job @iamigo